workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityWorkerMenuGenerator.tt

```diff
@@ -62,21 +62,85 @@ var workerType = workerTypeList[i];
             WorkerBuilder.Clean();
             Debug.Log("Clean completed");
         }
-        
-        private static void MenuBuild(BuildEnvironment environment, params string[] platforms)
+
+        private static bool CheckWorkersCanBuildForEnvironment(BuildEnvironment environment, string[] workerTypes)
```

Comment:
Does this need to be generated?

Reply:
Nope, moving into the Checker class

Reply:
Feels a bit strange as it's only used within this menu, so I think I won't modify the PR further